torchinductor/scheduler.py

```diff
@@ -554,6 +554,8 @@ def draw_buffers(nodes, fname, print_graph=False):
         if "fusion_meta" not in node.meta:
             continue
         group = node.meta["fusion_meta"].group
+        if isinstance(group, tuple):
+            group = group[1]
```

Comment:
what's `group[1]`?

Reply:
It's the "non-device" part of group. Group looks like (device, indexing_info).